records/state/state_test.go

```diff
@@ -1,11 +1,13 @@
-package state
+package state_test
```

Comment:
that's pretty interesting. i'm wondering if I can use this in the k8s-mesos "km" main package since it looks like the _test.go stuff in package main is bleeding into the generated non-test executable.